sdk/identity/Azure.Identity/src/Credentials/AzureCliCredential.cs

```diff
@@ -160,6 +162,11 @@ private async ValueTask<AccessToken> RequestCliAccessTokenAsync(bool async, Toke
                 throw new AuthenticationFailedException($"{AzureCliFailedError} {Troubleshoot} {exception.Message}");
             }
 
+            if (_logAccountDetails)
+            {
+                AzureIdentityEventSource.Singleton.AuthenticatedAccountDetails(null, _tenantId, null, null);
```

Comment:
Did we want to add a helper method to try to pull the authentication details out of the access token if possible?

Reply:
I think we discussed adding it after the fact, but I'm good with doing it now.